eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -69,6 +64,11 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 @ExtendWith(OnlineCondition.class)
 final class ProbeMojoTest {
+    /**
+     * Fake hashes.
+     */
+    private static final Map<String, CommitHash> HASHES = new CommitHashesMap.Fake();
```

Comment:
@maxonfjvipon [personal opinion] It is potential concurrency problem. All our tests work in parallel, so they can fulfil `HASHES` simultaneously. As I understand `CommitHashesMap.Fake` is thread-safe, at least for now, but we can change it in the future, which can lead to some problems.
Moreover the benefit which you get from the `static` field is not significant.
___
Of course, you can argue, that in the future we can make `CommitHashesMap.Fake` thread-safe,  so I leave it up to you.